tests/e2e/features/artifacts/transferPage/atrifacts-transfers.feature

```diff
@@ -147,3 +147,34 @@ Feature: Artifacts - UI
     Then Element with "text" "ZZ" should be "visible"
     Then Element with "text" "0x1ab721...184" should be "visible"
     Then Element with "partial src" "zz.png" should be "visible"
+
+  @id1537 @id1404
+  Scenario: Check artifacts on Transaction submitted pop up/Transaction completed pop up (Transfer)
+    Given I am on the Main page
+    When I go to page "/transaction/zksync/era/send?network=era-goerli&address=0x2CF4F78917A16C9584AeB5d4c5bD2713d724C75d"
+    When I choose "ETH" as token and insert "0.0000000001" as amount
+    When I "confirm" transaction after clicking "Send to zkSync Era Testnet" button
+    #Transaction submitted pop up - id1537
+    Then Message "Transaction submitted" should be visible
+    Then Element with "partial class" "progress-plane-animation" should be "visible"
+    Then Element with "text" " Track status " should be "visible"
+    #Transaction completed pop up - id1404
+    Then Message "Transaction completed" should be visible
+    #Success mark
+    Then Element with "partial class" "lottie-animation-container" should be "visible"
+    Then Element with "text" "Send" should be "visible"
+    #Time of tx
+    Then Element with "class" "button-line-body-info-underline" should be "visible"
+    Then Element with "text" "0.0000000001" should be "visible"
+    Then Modal card element with the "//*[text()='ETH']" xpath should be "visible"
+    #Token icon
+    Then Modal card element with the "//*[contains(@src, 'eth.svg?alt=media&token=1985e3d8-3aa7-4d04-8839-565d4c341615')]" xpath should be "visible"
```

Comment:
Does it possible to simplify this step somehow? is this Green success mark or what, to tell the truth difficult to understand.